src/ngraph/op/util/attr_types.hpp

```diff
@@ -166,12 +166,37 @@ namespace ngraph
             NONE = 0,
             EXPLICIT = NONE,
             NUMPY,
-            PDPD
+            PDPD,
         };
 
         NGRAPH_API
         std::ostream& operator<<(std::ostream& s, const AutoBroadcastType& type);
     }
+    namespace op
+    {
+        /// \brief BroadcastType specifies rules used for mapping of input tensor axes to output
```

Comment:
It would be nice to have some notes here for the need to have 2 similar enums, how they differ and where to use? Else it will be confusing to a user. There's a lot of discussion in the PR but once it is merged, it will not be easily accessible.

Reply:
I added short comment about it